connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSqlServerIntegrationSuite.scala

```diff
@@ -374,4 +375,58 @@ class MsSqlServerIntegrationSuite extends DockerJDBCIntegrationSuite {
     val filtered = df.where(col("c") === 0).collect()
     assert(filtered.length == 0)
   }
+
+  test("prepareQuery and query JDBC options") {
```

Comment:
Could you put the jira number here?

Reply:
Added in https://github.com/apache/spark/pull/36440/commits/aca8fa236f2de5144eb68e970f3b00255433ab7b